geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/InternalLocatorIntegrationTest.java

```diff
@@ -137,6 +137,18 @@ public void startedLocatorHasLocator() throws IOException {
     assertThat(InternalLocator.hasLocator()).isTrue();
   }
 
+  @Test
```

Comment:
I'm not saying this test is invalid, but it doesn't validate the fix below in `InternalLocator` i.e. if one comments out that code the test still passes...

Reply:
true, but I couldn't see how to write a better test.  Maybe you'd like to work with me on that?

Reply:
I think that, in fact, the test is invalid.  I'll post a better one.